prisma/migrations/20230626113518_add_batch_payment_table/migration.sql

```diff
@@ -0,0 +1,17 @@
+-- CreateTable
+CREATE TABLE "batch_payments" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "transaction_hash" TEXT NOT NULL,
+    "recipients_count" INTEGER NOT NULL,
+    "total_amount" DOUBLE PRECISION NOT NULL,
+    "group_id" UUID NOT NULL,
```

Comment:
We don't need to have `group_id` there, because batch payments relate to selected people, not a group. Instead we've to create a new table named `batch_payment_recipients` and set relation between `batch_payments_recipients.batch_payment_id` and `batch_payments.id`